### PR TITLE
Create initial models

### PR DESCRIPTION
**Pull request by randyxli**

### CODE REVIEW

File: listings/models.py [hunk 1]
```diff
@@ -1,3 +1,15 @@
 from django.db import models
+from django.contrib.auth.models import User
 
-# Create your models here.
+class Listing(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
```
Comment (jaypatel11) [inline listings/models.py:5]: Shouldn't ids be part of listing and user model? From a db query perspective, having access to id should be helpful

Comment (randyxli) [inline listings/models.py:5]: Thanks for the comment Jay. The `id` field is added automatically to all models, according to https://docs.djangoproject.com/en/1.9/topics/db/models/

Comment (jaypatel11) [inline listings/models.py:5]: Oh awesome! I didn't know that.

**Comment by jaypatel11** [comment]
+1
This issue can be closed too since both user and listings model are now created
https://github.com/randyxli/mplant/issues/10

**Pull request merged**